gpt4all-chat/src/chatlistmodel.cpp

```diff
@@ -193,12 +198,12 @@ void ChatsRestoreThread::run()
             // Read the version
             qint32 version;
             in >> version;
-            if (version < 1) {
+            if (version < 1 || version > CHAT_FORMAT_VERSION) {
```

Comment:
I'd prefer a more robust message for the case where version > CHAT_FORMAT_VERSION indicating the chat came from a more recent version than is currently running.

Reply:
Changed.